docs/learn.md

```diff
@@ -400,7 +400,318 @@ over the way these JavaScript packages are downloaded and bundled.
+`melange.emit` stanzas. In these cases, it is useful to have a way to build
+individual stanzas. To do so, one can define explicit aliases for each one of
+them by using the `alias` field.
+
+Let’s define a custom alias `app` for our `melange.emit` stanza: 
+
+```bash
+(melange.emit
+ (target app)
+ (alias app)
+ (libraries lib))
+```
+
+Now, when building with Dune, we can refer to this new alias:
+
+```bash
+$ dune build @app
+```
+
+Note that if we try to build again using the default `melange` alias, Dune will
+return an error, as there are no more targets attached to it.
+
+```bash
+$ dune build @melange
+Error: Alias "melange" specified on the command line is empty.
+It is not defined in . or any of its descendants.
+```
+
+#### Handling assets
+
+The last topic we will go through in this demo project is asset handling.
+Sometimes we want to use CSS files, fonts, or other assets in our Melange
+projects. Due to the way Dune works, our assets will have to be copied to the
+`_build` folder as well. To make this process as easy as possible, the Melange
+integration with Dune provides two ways to do this:
+
+- For `library` stanzas, a field `melange.runtime_deps`
+- For `melange.emit` stanzas, a field `runtime_deps`
+
+Both fields are documented in the [Melange
+page](https://dune.readthedocs.io/en/latest/melange.html) of the Dune
+documentation site.
+
+For the sake of learning how to work with assets in a Melange project, let’s say
+that we want to read the string in `Lib.name` from a text file. We will combine
+the field `melange.runtime_deps` with some bindings to Node that Melange
+provides. Check the next section, ["Communicate with
+JavaScript"](#communicate-with-javascript), it you want to learn more about how
+bindings work.
+
+So, let’s add a new file `name.txt` inside `lib` folder, that just contains the
+name `Jane`.
+
+Then, adapt the `lib/dune` file. We will need to add the `melange.runtime_deps`
+field, as well as a [`preprocessing`
+field](https://dune.readthedocs.io/en/stable/reference/preprocessing-spec.html)
+that will allow to use the `bs.raw` extension, in order to get the value of the
+`__dirname` environment variable:
+
+```bash
+(library
+ (name lib)
+ (modes melange)
+ (melange.runtime_deps name.txt)
+ (preprocess (pps melange.ppx)))
+```
+
+Finally, update `lib/lib.ml` to read from the recently added file:
+
+```ocaml
+let dir = [%bs.raw "__dirname"]
+let file = "name.txt"
+let name = Node.Fs.readFileSync (dir ^ "/" ^ file) `ascii
+```
+
+After these changes, once we build the project, we should still be able to run
+the application file with Node:
+
+```bash
+$ dune build @app
+$ node _build/default/app/app.js
+Jane
+```
+
+The same approach could be used to copy fonts, CSS or SVG files, or any other
+asset in your project. Note that Dune offers great flexibility to copy runtime
+assets using wildcards or globs, so one can simplify the configuration when
```

Comment:
let's add a link to the dune docs on "dependency specification", not just globs.
https://dune.readthedocs.io/en/stable/concepts.html#dependency-specification

Reply:
https://github.com/melange-re/documentation-site/pull/25/commits/d2d429b270fbc284f929b8d292410c531dbfc6d0